cyprecice/cyprecice.pyx

```diff
@@ -1193,6 +1193,81 @@ cdef class Interface:
         self.thisptr.readScalarData (data_id, vertex_id, _value)
         return _value
 
+    def set_mesh_access_region (self, mesh_id, bounding_box):
+        """
+        EXPERIMENTAL
+        This function is required if you don't want to use the mapping schemes in preCICE, but rather 
+        want to use your own solver for data mapping. As opposed to the usual preCICE mapping, only a
+        single mesh (from the other participant) is now involved in this situation since an 'own' 
+        mesh defined by the participant itself is not required any more. In order to re-partition the
+        received mesh, the participant needs to define the mesh region it wants read data from and 
+        write data to. The mesh region is specified through an axis-aligned bounding box given by the
+        lower and upper [min and max] bounding-box limits in each space dimension [x, y, z].
+
+        Parameters
+        ----------
+        mesh_id : int
+            ID of the mesh you want to access through the bounding box
+        bounding_box : array_like
+            Axis aligned bounding boxes which has in 3D the format 
+            [x_min, x_max, y_min, y_max, z_min, z_max]
```

Comment:
Why not the same format as for vertices? So: `n x d`, where `n = 2`, because you have two vertices `v_min` and `v_max`

Reply:
I got a bit carried away here: https://github.com/precice/precice/issues/1111